src/shared/config.js

```diff
@@ -44,3 +44,7 @@ export const NODE_REQUEST_TIMEOUT = 6000 * 2;
 export const DEFAULT_RETRIES = 4;
 
 export const IRI_API_VERSION = '1';
+
+export const QUORUM_THRESHOLD = 67;
+export const QUORUM_NODES_SIZE = 7;
```

Comment:
I think we can change the variable names here
```suggestion
export const QUORUM_SIZE = 7;
```

Reply:
Resolved [here](https://github.com/iotaledger/trinity-wallet/pull/631/commits/d8ad41e6a738ac82b528e3fc7ef87ee6225b956b#diff-78157a2166aa6eaa7e812e543c8729ddR15)